app/src/main/java/com/kickstarter/ui/fragments/PledgeFragment.kt

```diff
@@ -186,7 +193,12 @@ class PledgeFragment : BaseFragment<PledgeFragmentViewModel.ViewModel>(), Reward
         reward_snapshot.pivotX = 0f
         reward_snapshot.pivotY = 0f
 
-        val rewardViewHolder = HorizontalRewardViewHolder(reward_to_copy, null)
+        val rewardViewHolder = when {
+            RewardUtils.isNoReward(reward) -> HorizontalNoRewardViewHolder(no_reward_layout, null)
+            else -> HorizontalRewardViewHolder(reward_layout, null)
+        }
+
+        rewardViewHolder.itemView.visibility = View.VISIBLE
```

Comment:
Since we always have the No Reward view holder, wouldn't it be always visible? (Maybe we can set its visibility on the drawable?)

Reply:
`rewardViewHolder.itemView` is either `no_reward_layout` or `reward_layout` and both of their initial visibility states are `INVISIBLE`